pkg/util/label_test.go

```diff
@@ -249,7 +281,9 @@ func TestRemoveLabel(t *testing.T) {
 						},
 						"spec": map[string]interface{}{
 							"replicas": 2,
-						}}},
```

Comment:
The original writing is a style and, I feel it do not need be modified.

Reply:
But in some positions, it was different -
https://github.com/karmada-io/karmada/blob/b4b6d692ca963c3d53de4a61077f4380613be0a5/pkg/util/label_test.go#L161-L166
which is why I changed it. Should I change this back to the original?

Reply:
I think it's fine to keep the status, it's not something that needs to be corrected.